src/dialects/postgres/query-generator.js

```diff
@@ -295,15 +295,20 @@ export class PostgresQueryGenerator extends AbstractQueryGenerator {
     return super.handleSequelizeMethod.call(this, smth, tableName, factory, options, prepend);
   }
 
-  addColumnQuery(table, key, attribute) {
+  addColumnQuery(table, key, attribute, options = {}) {
     const dbDataType = this.attributeToSQL(attribute, { context: 'addColumn', table, key });
     const dataType = attribute.type || attribute;
     const definition = this.dataTypeMapping(table, key, dbDataType);
     const quotedKey = this.quoteIdentifier(key);
     const quotedTable = this.quoteTable(this.extractTableDetails(table));
-    const checkColumnExists = attribute.checkColumnExists;
+    let ifNotExists = options.ifNotExists;
```

Comment:
Avoid reassigning a variable if it's not necessary (especially if what it contains changes meaning). Prefer this:

```
const ifNotExists = options.ifNotExists ? 'IF NOT EXISTS ' : '';
```